devices/xiaomi.js

```diff
@@ -114,9 +115,23 @@ const fzLocal = {
                 case 0x0275:
                     result['valve_alarm'] = {1: true, 0: false}[value];
                     break;
+                case 247: {
+                    const heartbeat = xiaomiTrvUtils.decodeTrvHeartbeat(meta, model, value);
+
+                    meta.logger.debug(`${model.zigbeeModel}: Processed heartbeat message into payload ${JSON.stringify(heartbeat)}`);
+
+                    // Overwrite device firmware default version with actual version
+                    if (heartbeat.firmware_version) {
+                        // TODO How to trigger update in UI?
+                        meta.device.softwareBuildID = heartbeat.firmware_version;
```

Comment:
@Koenkk is there any way to trigger an update of the UI, e.g. programmatic reconfiguration? With the current implementation the firmware version label only updates after a restart or reconfiguration. Unfortunately this in not directly available/accessible in `configure` because the heartbeat message cannot be requested on demand.

Reply:
Is the `softwareBuildID` different from this `firmware_version`?

Reply:
Yes, as written in the PR description.

Reply:
The 'firmware_version` here is the _real_ version that is actually installed.

Reply:
Currently this is not possible and it will require some work to get this working. I guess you are talking about the `softwareBuildID` in the frontend which comes from `bridge/devices` (for this https://github.com/Koenkk/zigbee2mqtt/blob/bb3e8f6d414f126cd7f57686e0e9798f285eb78e/lib/extension/bridge.ts#L612 has to be called).

Since firmware updating is not done frequently, I feel it acceptable to state in the docs that you have to restart z2m some time after the update to get the actual version.

Reply:
I updated the edge with this commit, the latest firmware version is showing up fine, but the OTA keeps offering to update to the new version, is there any way to cancel this so that the OTA is not offered?